app/views/layouts/application.html.erb

```diff
@@ -37,54 +36,69 @@
-              <%= image_tag 'img/icon-dot-gov.svg', alt: 'Dot gov', class: 'usa-banner__icon usa-media-block__img' %>
-              <div class="usa-media-block__body">
-                <p>
-                  <strong>The .gov means it’s official.</strong>
-                  <br>
-                  Federal government websites often end in .gov or .mil. Before sharing sensitive information, make sure you’re on a federal government site.
-                </p>
+          <div class="usa-banner__content usa-accordion__content" id="gov-banner">
+            <%= javascript_tag(nonce: true) do %>
+              document.getElementById('gov-banner').setAttribute('hidden', '');
+            <% end %>
+            <div class="grid-row grid-gap-lg">
+              <div class="usa-banner__guidance tablet:grid-col-6">
+                <%= image_tag 'icon-dot-gov.svg', alt: 'Dot gov', class: 'usa-banner__icon usa-media-block__img', role: 'img', aria: { hidden: true } %>
+                <div class="usa-media-block__body">
+                  <p>
+                    <strong>Official websites use .gov</strong>
+                    <br>
+                    A <strong>.gov</strong> website belongs to an official government organization in the United States.
+                  </p>
+                </div>
               </div>
-            </div>
-            <div class="usa-banner__guidance tablet:grid-col-6">
-              <%= image_tag 'img/icon-https.svg', alt: 'Https', class: 'usa-banner__icon usa-media-block__img' %>
-              <div class="usa-media-block__body">
-                <p>
-                  <strong>The site is secure.</strong>
-                  <br>
-                  The <strong>https://</strong> ensures that you are connecting to the official website and that any information you provide is encrypted and transmitted securely.
-                </p>
+              <div class="usa-banner__guidance tablet:grid-col-6">
+                <%= image_tag 'icon-https.svg',
+                  alt: '',
+                  class: 'usa-banner__icon usa-media-block__img',
+                  role: 'img',
+                  aria: { hideen: true } %>
+                <div class="usa-media-block__body">
+                  <p>
+                    <strong>Secure .gov websites use HTTPS</strong>
+                    <br>
+                    A <strong>lock</strong> (
+                      <%= image_tag 'lock.svg',
+                        alt: 'A locked padlock',
+                        class: 'usa-banner__lock-image',
+                        role: 'img' %>
+                    ) or <strong>https://</strong> means you’ve safely connected to the .gov website. Share sensitive information only on official, secure websites.
+                    The <strong>https://</strong> ensures that you are connecting to the official website and that any information you provide is encrypted and transmitted securely.
```

Comment:
I think this second sentence should be removed? I think it's what's being updated.

```suggestion
```